common-rendering/src/editorialPalette/text.ts

```diff
@@ -419,6 +419,10 @@ const headlineTag = (_format: ArticleFormat): Colour => neutral[100];
 const headlineTagDark = (_format: ArticleFormat): Colour => neutral[10];
 
 const follow = (format: ArticleFormat): Colour => {
+	if (format.design === ArticleDesign.Gallery) {
```

Comment:
given that we switch on `format.design` below on line 428, it would be nice if there was a way to bring this check for `ArticleDesign.Gallery` into the main switch-case block. Having said that, I can't think of a clean way to do so! Just thought I'd flag in case you can think of a good way of doing it :)